toolbox/gpu-ml-benchmarks/deploy_nvidiadl_ssd.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
```

Comment:
could you check for "no flag" (good) / `-h` or --help` / "flags" (fail)

is there nothing interesting to configure for the benchmarking execution? (that's quite possible)
how long does it take to run the benchmark in it current configuration?